pkg/git/providers/git/git.go

```diff
@@ -0,0 +1,48 @@
+package git
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+const (
+	EksaGitPasswordTokenEnv   = "EKSA_GIT_PASSWORD"
+	EksaGitPrivateKeyTokenEnv = "EKSA_GIT_PRIVATE_KEY"
+)
+
+type Options struct {
+	RepositoryUrl string
+	User          string
+}
+
+func GetGitPasswordFromEnv() (string, error) {
+	password, err := ReadEnvVar(EksaGitPasswordTokenEnv)
+	if err != nil {
+		return "", err
+	}
+
+	return password, nil
```

Comment:
If you are not going to add context to the error, you can just
```suggestion
	return ReadEnvVar(EksaGitPasswordTokenEnv)
```